docker/docker_image_src.go

```diff
@@ -129,9 +129,42 @@ func (s *dockerImageSource) ensureManifestIsLoaded() error {
 	return nil
 }
 
+func getExternalBlob(urls []string) (io.ReadCloser, int64, error) {
+	var (
+		resp *http.Response
+		err  error
+	)
+	for _, url := range urls {
+		resp, err = http.Get(url)
```

Comment:
we should use the previously setup docker client to comply with https://github.com/projectatomic/skopeo/issues/250